tests/integration/components/donation/donation-container-test.js

```diff
@@ -1,35 +1,73 @@
 import { moduleForComponent, test } from 'ember-qunit';
+import donationContainerComponent from '../../../pages/components/donation/donation-container';
 import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+import stubService from 'code-corps-ember/tests/helpers/stub-service';
+import setHandler from 'code-corps-ember/tests/helpers/set-handler';
+
+let page = PageObject.create(donationContainerComponent);
 
 moduleForComponent('donation/donation-container', 'Integration | Component | donation/donation container', {
-  integration: true
+  integration: true,
+  beforeEach() {
+    setHandler(this, 'donate');
+    page.setContext(this);
+  },
+  afterEach() {
+    page.removeContext();
+  }
 });
 
-test('it renders no project error when no project is passed in', function(assert) {
-  assert.expect(1);
+test('it renders correctly', function(assert) {
+  assert.expect(3);
 
-  this.render(hbs`{{donation/donation-container}}`);
+  this.set('amount', 100);
+  this.set('projectTitle', 'CodeCorps');
 
-  let mainContent = this.$().find('p').eq(0);
-  assert.equal(mainContent.text().trim(), 'No project selected.');
+  page.render(hbs`{{donation/donation-container donate=(action donateHandler) donationAmount=amount projectTitle=projectTitle}}`);
+
+  assert.equal(page.donationAmountText, '$100.00 given each month', 'Amount text is rendered correctly');
+  assert.equal(
+    page.paymentInformationText,
+    'Your payment method will be charged $100.00 per month to support CodeCorps.',
+    'Payment information renders correctly.'
+  );
+  assert.ok(page.creditCardIsRendered, 'Credit card component is rendered.');
 });
 
-test('it renders donation amount and frequency', function(assert) {
+test('it handles clicking card submit button correctly', function(assert) {
```

Comment:
@JoshSmith I'm questioning what we are testing here.  Most of the actions in this particular test are for the credit card component.  This test fails because of the submit action on `donation/credit-card` (the child component).  I think the test coverage for submitting should be handled there and we should only worry about rendering what is appropriate here, as this is really just the container of the components that hold the logic.